config/application_template.properties

```diff
@@ -2,6 +2,16 @@ spring.datasource.driverClassName=com.mysql.jdbc.Driver
 spring.datasource.url=jdbc:mysql://server_url/cpm?
 spring.datasource.username=user
 spring.datasource.password=password
-
 spring.jpa.database-platform=org.hibernate.dialect.MySQLInnoDBDialect
 spring.jpa.show-sql=true
+
+ctools.direct.url=
```

Comment:
For now, all properties are inside one file. Will separate the security properties in the future.